CRITICAL - 2024-11-03 21:44:32 --> ParseError: syntax error, unexpected single-quoted string "register", expecting "function" or "const"
[Method: POST, Route: register/store]
in APPPATH\Config\Validation.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Config\\Validation.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Validation')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Validation')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Validation')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Validation')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Validation', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Validation')
 9 SYSTEMPATH\Config\Services.php(833): config('Config\\Validation')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::validation(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('validation', [...])
12 SYSTEMPATH\Config\Services.php(830): CodeIgniter\Config\BaseService::getSharedInstance('validation', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::validation()
14 APPPATH\Controllers\RegisterController.php(18): CodeIgniter\Config\BaseService::__callStatic('validation', [])
15 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
16 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
17 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
18 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
19 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
20 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 21:48:23 --> ParseError: syntax error, unexpected single-quoted string "register", expecting "function" or "const"
[Method: POST, Route: register/store]
in APPPATH\Config\Validation.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Config\\Validation.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Validation')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Validation')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Validation')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Validation')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Validation', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Validation')
 9 SYSTEMPATH\Config\Services.php(833): config('Config\\Validation')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::validation(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('validation', [...])
12 SYSTEMPATH\Config\Services.php(830): CodeIgniter\Config\BaseService::getSharedInstance('validation', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::validation()
14 APPPATH\Controllers\RegisterController.php(18): CodeIgniter\Config\BaseService::__callStatic('validation', [])
15 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
16 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
17 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
18 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
19 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
20 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 21:48:41 --> ParseError: syntax error, unexpected single-quoted string "register", expecting "function" or "const"
[Method: POST, Route: register/store]
in APPPATH\Config\Validation.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Config\\Validation.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Validation')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Validation')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Validation')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Validation')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Validation', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Validation')
 9 SYSTEMPATH\Config\Services.php(833): config('Config\\Validation')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::validation(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('validation', [...])
12 SYSTEMPATH\Config\Services.php(830): CodeIgniter\Config\BaseService::getSharedInstance('validation', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::validation()
14 APPPATH\Controllers\RegisterController.php(18): CodeIgniter\Config\BaseService::__callStatic('validation', [])
15 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
16 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
17 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
18 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
19 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
20 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 21:49:45 --> ErrorException: Undefined property: Config\Validation::$ruleSets
[Method: POST, Route: register/store]
in SYSTEMPATH\Validation\Validation.php on line 119.
 1 SYSTEMPATH\Validation\Validation.php(119): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: Config\\Validation::$ruleSets', 'C:\\xampp\\htdocs\\Proyectos\\pp_2024\\system\\Validation\\Validation.php', 119)
 2 SYSTEMPATH\Config\Services.php(835): CodeIgniter\Validation\Validation->__construct(Object(Config\Validation), Object(CodeIgniter\View\View))
 3 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::validation(Object(Config\Validation), false)
 4 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('validation', [...])
 5 SYSTEMPATH\Config\Services.php(830): CodeIgniter\Config\BaseService::getSharedInstance('validation', null)
 6 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::validation()
 7 APPPATH\Controllers\RegisterController.php(18): CodeIgniter\Config\BaseService::__callStatic('validation', [])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 21:50:05 --> ErrorException: Undefined property: Config\Validation::$ruleSets
[Method: POST, Route: register/store]
in SYSTEMPATH\Validation\Validation.php on line 119.
 1 SYSTEMPATH\Validation\Validation.php(119): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: Config\\Validation::$ruleSets', 'C:\\xampp\\htdocs\\Proyectos\\pp_2024\\system\\Validation\\Validation.php', 119)
 2 SYSTEMPATH\Config\Services.php(835): CodeIgniter\Validation\Validation->__construct(Object(Config\Validation), Object(CodeIgniter\View\View))
 3 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::validation(Object(Config\Validation), false)
 4 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('validation', [...])
 5 SYSTEMPATH\Config\Services.php(830): CodeIgniter\Config\BaseService::getSharedInstance('validation', null)
 6 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::validation()
 7 APPPATH\Controllers\RegisterController.php(18): CodeIgniter\Config\BaseService::__callStatic('validation', [])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-03 21:51:27 --> mysqli_sql_exception: Unknown column 'email' in 'where clause' in C:\xampp\htdocs\Proyectos\pp_2024\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\Proyectos\pp_2024\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\Proyectos\pp_2024\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\Proyectos\pp_2024\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\Proyectos\pp_2024\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Proyectos\pp_2024\system\Validation\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Proyectos\pp_2024\system\Validation\Validation.php(339): CodeIgniter\Validation\Rules->is_unique('prueba123@gmail...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\Proyectos\pp_2024\system\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('email', 'email', 'prueba123@gmail...', Array, Array, 'email')
#7 C:\xampp\htdocs\Proyectos\pp_2024\app\Controllers\RegisterController.php(26): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\Proyectos\pp_2024\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 C:\xampp\htdocs\Proyectos\pp_2024\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 C:\xampp\htdocs\Proyectos\pp_2024\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Proyectos\pp_2024\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Proyectos\pp_2024\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\Proyectos\pp_2024\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-11-03 21:51:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'email' in 'where clause'
[Method: POST, Route: register/store]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(339): CodeIgniter\Validation\Rules->is_unique('prueba123@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('email', 'email', 'prueba123@gmail.com', [...], [...], 'email')
 5 APPPATH\Controllers\RegisterController.php(26): CodeIgniter\Validation\Validation->run()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 21:51:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'email' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuarios`
WHERE `email` = \'prueba123@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuarios`
WHERE `email` = \'prueba123@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(339): CodeIgniter\Validation\Rules->is_unique('prueba123@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('email', 'email', 'prueba123@gmail.com', [...], [...], 'email')
 7 APPPATH\Controllers\RegisterController.php(26): CodeIgniter\Validation\Validation->run()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 21:51:27 --> [Caused by] mysqli_sql_exception: Unknown column 'email' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT 1
FROM `usuarios`
WHERE `email` = \'prueba123@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuarios`
WHERE `email` = \'prueba123@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuarios`
WHERE `email` = \'prueba123@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuarios`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(339): CodeIgniter\Validation\Rules->is_unique('prueba123@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('email', 'email', 'prueba123@gmail.com', [...], [...], 'email')
 8 APPPATH\Controllers\RegisterController.php(26): CodeIgniter\Validation\Validation->run()
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 22:17:33 --> ErrorException: Undefined property: Config\Validation::$templates
[Method: POST, Route: register/store]
in SYSTEMPATH\Validation\Validation.php on line 687.
 1 SYSTEMPATH\Validation\Validation.php(687): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: Config\\Validation::$templates', 'C:\\xampp\\htdocs\\Proyectos\\pp_2024\\system\\Validation\\Validation.php', 687)
 2 APPPATH\Views\register.php(13): CodeIgniter\Validation\Validation->listErrors()
 3 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Views\\register.php')
 4 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('register', [], true)
 6 APPPATH\Controllers\RegisterController.php(27): view('register', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-03 22:26:28 --> mysqli_sql_exception: Unknown column 'username' in 'field list' in C:\xampp\htdocs\Proyectos\pp_2024\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\Proyectos\pp_2024\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\Proyectos\pp_2024\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\Proyectos\pp_2024\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\Proyectos\pp_2024\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\Proyectos\pp_2024\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Proyectos\pp_2024\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Proyectos\pp_2024\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\Proyectos\pp_2024\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\Proyectos\pp_2024\app\Controllers\RegisterController.php(32): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\Proyectos\pp_2024\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#10 C:\xampp\htdocs\Proyectos\pp_2024\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 C:\xampp\htdocs\Proyectos\pp_2024\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\Proyectos\pp_2024\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\Proyectos\pp_2024\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\Proyectos\pp_2024\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-11-03 22:26:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'username' in 'field list'
[Method: POST, Route: register/store]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`username`, `email`, `password`, `rol`) VALUES (:username:, :email:, :password:, :rol:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 22:26:28 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'username' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`username`, `email`, `password`, `rol`) VALUES (\'Juan\', \'juan@gmail.com\', \'$2y$10$K8yccmuB6Xxw.jew0BB.I.IP2Btg0mqN6Jq7xJU5jYYoQ./sL/Xo6\', \'estudiante\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`username`, `email`, `password`, `rol`) VALUES (\'Juan\', \'juan@gmail.com\', \'$2y$10$K8yccmuB6Xxw.jew0BB.I.IP2Btg0mqN6Jq7xJU5jYYoQ./sL/Xo6\', \'estudiante\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`username`, `email`, `password`, `rol`) VALUES (:username:, :email:, :password:, :rol:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 22:26:28 --> [Caused by] mysqli_sql_exception: Unknown column 'username' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `usuarios` (`username`, `email`, `password`, `rol`) VALUES (\'Juan\', \'juan@gmail.com\', \'$2y$10$K8yccmuB6Xxw.jew0BB.I.IP2Btg0mqN6Jq7xJU5jYYoQ./sL/Xo6\', \'estudiante\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`username`, `email`, `password`, `rol`) VALUES (\'Juan\', \'juan@gmail.com\', \'$2y$10$K8yccmuB6Xxw.jew0BB.I.IP2Btg0mqN6Jq7xJU5jYYoQ./sL/Xo6\', \'estudiante\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`username`, `email`, `password`, `rol`) VALUES (\'Juan\', \'juan@gmail.com\', \'$2y$10$K8yccmuB6Xxw.jew0BB.I.IP2Btg0mqN6Jq7xJU5jYYoQ./sL/Xo6\', \'estudiante\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`username`, `email`, `password`, `rol`) VALUES (:username:, :email:, :password:, :rol:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-11-03 22:35:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-03 22:45:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-03 23:09:26 --> ParseError: Unclosed '{' on line 7
[Method: GET, Route: login]
in APPPATH\Controllers\AuthController.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Controllers\\AuthController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\AuthController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\AuthController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\AuthController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-11-03 23:11:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-03 23:11:38 --> ParseError: Unclosed '{' on line 7
[Method: GET, Route: login]
in APPPATH\Controllers\AuthController.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Controllers\\AuthController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\AuthController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\AuthController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\AuthController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 23:21:52 --> ParseError: Unclosed '{' on line 7
[Method: GET, Route: login]
in APPPATH\Controllers\AuthController.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Controllers\\AuthController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\AuthController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\AuthController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\AuthController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-11-03 23:22:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-03 23:22:40 --> ParseError: Unclosed '{' on line 7
[Method: GET, Route: login]
in APPPATH\Controllers\AuthController.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Controllers\\AuthController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\AuthController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\AuthController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\AuthController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 23:26:30 --> ParseError: Unclosed '{' on line 14
[Method: GET, Route: login]
in APPPATH\Controllers\AuthController.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Controllers\\AuthController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\AuthController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\AuthController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\AuthController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 23:26:33 --> ParseError: Unclosed '{' on line 14
[Method: GET, Route: login]
in APPPATH\Controllers\AuthController.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Controllers\\AuthController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\AuthController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\AuthController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\AuthController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-11-03 23:27:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-03 23:27:33 --> ParseError: Unclosed '{' on line 14
[Method: GET, Route: login]
in APPPATH\Controllers\AuthController.php on line 44.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Controllers\\AuthController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\AuthController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\AuthController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\AuthController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 23:28:38 --> Error: Class "App\Controllers\Controller" not found
[Method: GET, Route: login]
in APPPATH\Controllers\AuthController.php on line 6.
 1 SYSTEMPATH\Autoloader\Autoloader.php(317): include_once()
 2 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Proyectos\\pp_2024\\app\\Controllers\\AuthController.php')
 3 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\AuthController')
 4 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\AuthController')
 5 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\AuthController', true)
 6 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-03 23:39:20 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: login]
in APPPATH\Controllers\AuthController.php on line 3.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
